Assets/Plugins/iOS/Shopify/Checkout/Checkout.WebViewController.swift

```diff
@@ -73,10 +75,32 @@ class WebViewController: UIViewController {
 // MARK: Actions
 extension Checkout.WebViewController {
     func loadCheckoutURL() {
-        let request = URLRequest(url: url)
+        let request = URLRequest(url: startURL)
         webView.load(request)
     }
     
+    func generateErrorPage(message: String, retryURL: URL) -> URL {
+        // Instead of using webView.loadHTMLString, we generate a new HTML file and store on disk so 
+        // we can use webView.loadFileURL which allows us differentiate between an error page and a 
+        // web page. Using webView.loadHTMLString will set the url to about:blank which isn't strong enough
+        // to tell us that it's not an error page.
+        let errorPageFileURL = Bundle.main.url(forResource: "errorPage", withExtension: "html")!
+        let polarisCSSFileURL = Bundle.main.url(forResource: "polaris-1.0.2", withExtension: "css")!
+        
+        let errorPageString = try! String(contentsOf: errorPageFileURL)
+        let polarisCSSString = try! String(contentsOf: polarisCSSFileURL)
```

Comment:
Can we store these resources, so that they are not read and loaded every single time we `generateErrorPage`?